{"level":"error","message":"3/13/2024---message: Configuration property \"expressWhiteList\" is not defined\nstatus: undefined\nstack: Error: Configuration property \"expressWhiteList\" is not defined\n    at Config.get (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/config@3.3.11/node_modules/config/lib/config.js:182:11)\n    at origin (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:4300:41)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/cors@2.8.5/node_modules/cors/lib/index.js:219:13\n    at optionsCallback (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/cors@2.8.5/node_modules/cors/lib/index.js:199:9)\n    at corsMiddleware (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/cors@2.8.5/node_modules/cors/lib/index.js:204:7)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:328:13)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:286:9\n    at Function.process_params (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:280:10)"}
{"level":"error","message":"3/13/2024---message: Configuration property \"expressWhiteList\" is not defined\nstatus: undefined\nstack: Error: Configuration property \"expressWhiteList\" is not defined\n    at Config.get (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/config@3.3.11/node_modules/config/lib/config.js:182:11)\n    at origin (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:4300:41)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/cors@2.8.5/node_modules/cors/lib/index.js:219:13\n    at optionsCallback (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/cors@2.8.5/node_modules/cors/lib/index.js:199:9)\n    at corsMiddleware (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/cors@2.8.5/node_modules/cors/lib/index.js:204:7)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:328:13)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:286:9\n    at Function.process_params (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:280:10)"}
{"level":"error","message":"3/13/2024---message: Configuration property \"expressWhiteList\" is not defined\nstatus: undefined\nstack: Error: Configuration property \"expressWhiteList\" is not defined\n    at Config.get (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/config@3.3.11/node_modules/config/lib/config.js:182:11)\n    at origin (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:4300:41)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/cors@2.8.5/node_modules/cors/lib/index.js:219:13\n    at optionsCallback (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/cors@2.8.5/node_modules/cors/lib/index.js:199:9)\n    at corsMiddleware (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/cors@2.8.5/node_modules/cors/lib/index.js:204:7)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:328:13)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:286:9\n    at Function.process_params (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:280:10)"}
{"level":"error","message":"3/13/2024---message: Configuration property \"expressWhiteList\" is not defined\nstatus: undefined\nstack: Error: Configuration property \"expressWhiteList\" is not defined\n    at Config.get (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/config@3.3.11/node_modules/config/lib/config.js:182:11)\n    at origin (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:4300:41)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/cors@2.8.5/node_modules/cors/lib/index.js:219:13\n    at optionsCallback (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/cors@2.8.5/node_modules/cors/lib/index.js:199:9)\n    at corsMiddleware (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/cors@2.8.5/node_modules/cors/lib/index.js:204:7)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:328:13)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:286:9\n    at Function.process_params (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:280:10)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3916:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3913:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3916:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3913:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3992:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3989:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3916:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3913:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3992:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3989:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3916:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3913:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3992:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3989:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3916:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3913:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3992:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3989:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3916:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3913:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3992:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3989:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3916:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3913:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3992:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3989:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3916:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3913:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3992:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3989:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at authMiddleware (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3688:46)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:175:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at authMiddleware (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3688:46)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:175:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at authMiddleware (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3688:46)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:175:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at authMiddleware (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3688:46)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:175:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at authMiddleware (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3688:46)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:175:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at authMiddleware (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3688:46)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:175:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at authMiddleware (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3688:46)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:175:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at authMiddleware (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3688:46)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:175:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at authMiddleware (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3688:46)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:175:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at authMiddleware (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3688:46)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/index.js:175:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"3/13/2024---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"12:40:32 PM---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"12:40:32 PM---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"12:40:32 PM---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"12:40:32 PM---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:41:06.156Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:41:06.183Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:41:06.285Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:41:06.298Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:41:13.771Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:41:13.803Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:41:17.547Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:41:17.573Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:41:17.643Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:41:17.663Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:42:26.567Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:42:26.595Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:42:26.647Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:42:26.661Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:42:27.982Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:42:28.032Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:42:28.033Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:42:28.102Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:42:28.114Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:44:07.247Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:44:07.292Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:44:07.360Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:44:07.374Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:44:55.691Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:44:55.721Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:44:57.020Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:44:57.046Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:11.388Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:11.394Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:11.760Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:11.804Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:12.999Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:13.051Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:18.176Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:18.215Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:18.298Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:18.304Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:28.185Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:28.234Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:31.890Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:31.937Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:36.828Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:36.863Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:47.710Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:47.716Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:47.730Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:47.738Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:53.430Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:45:53.464Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3962:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at refreshToken (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3959:50)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:53:47.454Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T10:55:10.931Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T11:14:57.713Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T11:16:14.089Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3661:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3884:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3881:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T11:59:10.179Z---message: You are not unauthorized\nstatus: 401\nstack: Error: You are not unauthorized\n    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3688:16)\n    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3911:40\n    at Generator.next (<anonymous>)\n    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75\n    at new Promise (<anonymous>)\n    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)\n    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3908:40)\n    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)"}
{"level":"error","message":"2024-03-13T11:59:10.221Z---message: \nInvalid `prisma.article.findMany()` invocation:\n\n{\n  skip: 0,\n  take: 10,\n  orderBy: [\n    {\n      : undefined\n    }\n  ],\n+ where: {\n+   AND: ArticleWhereInput | ArticleWhereInput[],\n+   OR: ArticleWhereInput[],\n+   NOT: ArticleWhereInput | ArticleWhereInput[],\n+   id: StringFilter | String,\n+   author: StringFilter | String,\n+   title: StringFilter | String,\n+   content: StringFilter | String,\n+   link: StringFilter | String\n+ }\n}\n\nArgument `where` must not be null.\nstatus: undefined\nstack: PrismaClientValidationError: \nInvalid `prisma.article.findMany()` invocation:\n\n{\n  skip: 0,\n  take: 10,\n  orderBy: [\n    {\n      : undefined\n    }\n  ],\n+ where: {\n+   AND: ArticleWhereInput | ArticleWhereInput[],\n+   OR: ArticleWhereInput[],\n+   NOT: ArticleWhereInput | ArticleWhereInput[],\n+   id: StringFilter | String,\n+   author: StringFilter | String,\n+   title: StringFilter | String,\n+   content: StringFilter | String,\n+   link: StringFilter | String\n+ }\n}\n\nArgument `where` must not be null.\n    at Cn (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3007:235)\n    at _n.handleRequestError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3207:9)\n    at _n.handleAndLogRequestError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3199:14)\n    at _n.<anonymous> (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3194:18)\n    at Generator.throw (<anonymous>)\n    at rejected (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:167:69)"}
{"level":"error","message":"2024-03-13T11:59:15.402Z---message: \nInvalid `prisma.article.findMany()` invocation:\n\n{\n  skip: 10,\n  take: 10,\n  orderBy: [\n    {\n      : undefined\n    }\n  ],\n+ where: {\n+   AND: ArticleWhereInput | ArticleWhereInput[],\n+   OR: ArticleWhereInput[],\n+   NOT: ArticleWhereInput | ArticleWhereInput[],\n+   id: StringFilter | String,\n+   author: StringFilter | String,\n+   title: StringFilter | String,\n+   content: StringFilter | String,\n+   link: StringFilter | String\n+ }\n}\n\nArgument `where` must not be null.\nstatus: undefined\nstack: PrismaClientValidationError: \nInvalid `prisma.article.findMany()` invocation:\n\n{\n  skip: 10,\n  take: 10,\n  orderBy: [\n    {\n      : undefined\n    }\n  ],\n+ where: {\n+   AND: ArticleWhereInput | ArticleWhereInput[],\n+   OR: ArticleWhereInput[],\n+   NOT: ArticleWhereInput | ArticleWhereInput[],\n+   id: StringFilter | String,\n+   author: StringFilter | String,\n+   title: StringFilter | String,\n+   content: StringFilter | String,\n+   link: StringFilter | String\n+ }\n}\n\nArgument `where` must not be null.\n    at Cn (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3007:235)\n    at _n.handleRequestError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3207:9)\n    at _n.handleAndLogRequestError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3199:14)\n    at _n.<anonymous> (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3194:18)\n    at Generator.throw (<anonymous>)\n    at rejected (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:167:69)"}
{"level":"error","message":"2024-03-13T11:59:15.964Z---message: \nInvalid `prisma.article.findMany()` invocation:\n\n{\n  skip: 20,\n  take: 10,\n  orderBy: [\n    {\n      : undefined\n    }\n  ],\n+ where: {\n+   AND: ArticleWhereInput | ArticleWhereInput[],\n+   OR: ArticleWhereInput[],\n+   NOT: ArticleWhereInput | ArticleWhereInput[],\n+   id: StringFilter | String,\n+   author: StringFilter | String,\n+   title: StringFilter | String,\n+   content: StringFilter | String,\n+   link: StringFilter | String\n+ }\n}\n\nArgument `where` must not be null.\nstatus: undefined\nstack: PrismaClientValidationError: \nInvalid `prisma.article.findMany()` invocation:\n\n{\n  skip: 20,\n  take: 10,\n  orderBy: [\n    {\n      : undefined\n    }\n  ],\n+ where: {\n+   AND: ArticleWhereInput | ArticleWhereInput[],\n+   OR: ArticleWhereInput[],\n+   NOT: ArticleWhereInput | ArticleWhereInput[],\n+   id: StringFilter | String,\n+   author: StringFilter | String,\n+   title: StringFilter | String,\n+   content: StringFilter | String,\n+   link: StringFilter | String\n+ }\n}\n\nArgument `where` must not be null.\n    at Cn (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3007:235)\n    at _n.handleRequestError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3207:9)\n    at _n.handleAndLogRequestError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3199:14)\n    at _n.<anonymous> (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3194:18)\n    at Generator.throw (<anonymous>)\n    at rejected (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:167:69)"}
{"level":"error","message":"2024-03-13T11:59:16.685Z---message: \nInvalid `prisma.article.findMany()` invocation:\n\n{\n  skip: 0,\n  take: 10,\n  orderBy: [\n    {\n      : undefined\n    }\n  ],\n+ where: {\n+   AND: ArticleWhereInput | ArticleWhereInput[],\n+   OR: ArticleWhereInput[],\n+   NOT: ArticleWhereInput | ArticleWhereInput[],\n+   id: StringFilter | String,\n+   author: StringFilter | String,\n+   title: StringFilter | String,\n+   content: StringFilter | String,\n+   link: StringFilter | String\n+ }\n}\n\nArgument `where` must not be null.\nstatus: undefined\nstack: PrismaClientValidationError: \nInvalid `prisma.article.findMany()` invocation:\n\n{\n  skip: 0,\n  take: 10,\n  orderBy: [\n    {\n      : undefined\n    }\n  ],\n+ where: {\n+   AND: ArticleWhereInput | ArticleWhereInput[],\n+   OR: ArticleWhereInput[],\n+   NOT: ArticleWhereInput | ArticleWhereInput[],\n+   id: StringFilter | String,\n+   author: StringFilter | String,\n+   title: StringFilter | String,\n+   content: StringFilter | String,\n+   link: StringFilter | String\n+ }\n}\n\nArgument `where` must not be null.\n    at Cn (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3007:235)\n    at _n.handleRequestError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3207:9)\n    at _n.handleAndLogRequestError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3199:14)\n    at _n.<anonymous> (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3194:18)\n    at Generator.throw (<anonymous>)\n    at rejected (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:167:69)"}
{"level":"error","message":"2024-03-13T11:59:19.872Z---message: \nInvalid `prisma.article.findMany()` invocation:\n\n{\n  skip: 0,\n  take: 10,\n  orderBy: [\n    {\n      : undefined\n    }\n  ],\n+ where: {\n+   AND: ArticleWhereInput | ArticleWhereInput[],\n+   OR: ArticleWhereInput[],\n+   NOT: ArticleWhereInput | ArticleWhereInput[],\n+   id: StringFilter | String,\n+   author: StringFilter | String,\n+   title: StringFilter | String,\n+   content: StringFilter | String,\n+   link: StringFilter | String\n+ }\n}\n\nArgument `where` must not be null.\nstatus: undefined\nstack: PrismaClientValidationError: \nInvalid `prisma.article.findMany()` invocation:\n\n{\n  skip: 0,\n  take: 10,\n  orderBy: [\n    {\n      : undefined\n    }\n  ],\n+ where: {\n+   AND: ArticleWhereInput | ArticleWhereInput[],\n+   OR: ArticleWhereInput[],\n+   NOT: ArticleWhereInput | ArticleWhereInput[],\n+   id: StringFilter | String,\n+   author: StringFilter | String,\n+   title: StringFilter | String,\n+   content: StringFilter | String,\n+   link: StringFilter | String\n+ }\n}\n\nArgument `where` must not be null.\n    at Cn (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3007:235)\n    at _n.handleRequestError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3207:9)\n    at _n.handleAndLogRequestError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3199:14)\n    at _n.<anonymous> (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3194:18)\n    at Generator.throw (<anonymous>)\n    at rejected (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:167:69)"}

    2024-03-13T12:04:38.592Z
    message: You are not unauthorized
    status: 401
    stack: Error: You are not unauthorized
    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3688:16)
    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3911:40
    at Generator.next (<anonymous>)
    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)
    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3908:40)
    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)
    

    2024-03-13T12:04:38.633Z
    message: 
Invalid `prisma.article.findMany()` invocation:

{
  skip: 0,
  take: 10,
  orderBy: [
    {
      : undefined
    }
  ],
+ where: {
+   AND: ArticleWhereInput | ArticleWhereInput[],
+   OR: ArticleWhereInput[],
+   NOT: ArticleWhereInput | ArticleWhereInput[],
+   id: StringFilter | String,
+   author: StringFilter | String,
+   title: StringFilter | String,
+   content: StringFilter | String,
+   link: StringFilter | String
+ }
}

Argument `where` must not be null.
    status: undefined
    stack: PrismaClientValidationError: 
Invalid `prisma.article.findMany()` invocation:

{
  skip: 0,
  take: 10,
  orderBy: [
    {
      : undefined
    }
  ],
+ where: {
+   AND: ArticleWhereInput | ArticleWhereInput[],
+   OR: ArticleWhereInput[],
+   NOT: ArticleWhereInput | ArticleWhereInput[],
+   id: StringFilter | String,
+   author: StringFilter | String,
+   title: StringFilter | String,
+   content: StringFilter | String,
+   link: StringFilter | String
+ }
}

Argument `where` must not be null.
    at Cn (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3007:235)
    at _n.handleRequestError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3207:9)
    at _n.handleAndLogRequestError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3199:14)
    at _n.<anonymous> (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3194:18)
    at Generator.throw (<anonymous>)
    at rejected (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:167:69)
    

    2024-03-13T12:06:07.998Z
    message: You are not unauthorized
    status: 401
    stack: Error: You are not unauthorized
    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3688:16)
    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3911:40
    at Generator.next (<anonymous>)
    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)
    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3908:40)
    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)
    

    2024-03-13T12:08:11.596Z
    message: You are not unauthorized
    status: 401
    stack: Error: You are not unauthorized
    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3688:16)
    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3911:40
    at Generator.next (<anonymous>)
    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)
    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3908:40)
    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)
    

    2024-03-13T12:10:23.815Z
    message: You are not unauthorized
    status: 401
    stack: Error: You are not unauthorized
    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3690:16)
    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3913:40
    at Generator.next (<anonymous>)
    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)
    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3910:40)
    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)
    

    2024-03-13T12:11:32.339Z
    message: You are not unauthorized
    status: 401
    stack: Error: You are not unauthorized
    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3689:16)
    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3912:40
    at Generator.next (<anonymous>)
    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)
    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3909:40)
    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)
    

    2024-03-13T12:12:38.394Z
    message: You are not unauthorized
    status: 401
    stack: Error: You are not unauthorized
    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3685:16)
    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3908:40
    at Generator.next (<anonymous>)
    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)
    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3905:40)
    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)
    

    2024-03-13T12:13:58.667Z
    message: You are not unauthorized
    status: 401
    stack: Error: You are not unauthorized
    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)
    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3896:40
    at Generator.next (<anonymous>)
    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)
    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3893:40)
    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)
    

    2024-03-13T12:18:16.308Z
    message: You are not unauthorized
    status: 401
    stack: Error: You are not unauthorized
    at Function.UnauthorizedError (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3673:16)
    at /Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3896:40
    at Generator.next (<anonymous>)
    at /Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:169:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:165:16)
    at me (/Users/a1/Documents/dev.it/app/dist/apps/backend/main.js:3893:40)
    at Layer.handle [as handle_request] (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/Users/a1/Documents/dev.it/app/node_modules/.pnpm/express@4.18.3/node_modules/express/lib/router/route.js:119:3)
    
